src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -518,21 +586,23 @@ $spacing: 4px;
 	&__label {
 		display: flex;
 		align-items: center;
+		flex-direction: row;
+		gap: 4px;
 		user-select: none;
 		min-height: $clickable-area;
 		border-radius: $clickable-area;
 		padding: 4px $icon-margin;
-		margin: 0 #{-$icon-margin};
+		// Set to 100% to make text overflow work on button style
+		width: 100%;
+		// but restrict to content so plain checkboxes / radio switches do not expand
+		max-width: fit-content;
```

Comment:
This needs to be overwritten for the `NcAppSidebarTabs` :see_no_evil: :laughing: 
![Screenshot 2023-05-11 at 19-32-53 Nextcloud Vue Style Guide](https://github.com/nextcloud/nextcloud-vue/assets/2496460/495a4887-7a04-40c0-ab72-877c29fb84ad)

But we are getting closer, I think :slightly_smiling_face:

Reply:
Fix that. Hopefully everything works now as expected :wink: